service/src/main/java/com/ge/predix/acs/attribute/readers/ExternalAttributeReader.java

```diff
@@ -120,7 +120,7 @@ CachedAttributes getAttributesFromAdapters(final String identifier) throws Attri
             try {
                 attributesResponse = this.getAdapterOauth2RestTemplate(adapterConnection).
                         getForEntity(adapterUrl, AttributesResponse.class).getBody();
-            } catch (final Throwable e) {
+            } catch (final Exception e) {
```

Comment:
You are right - based on 'Error' and 'Exception' documentation  this is one of the main differences between the two. So, yes lets not catch 'Throwable's going forward.